pkgs/by-name/kr/kryoflux/package.nix

```
@@ -0,0 +1,56 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  fetchurl,
+  makeWrapper,
+  jre,
+  fmt_9,
+  libusb1,
+}:
+let
+  archdir =
+    if stdenv.hostPlatform.isx86_64 then
+      "x86_64"
+    else if stdenv.hostPlatform.isAarch64 then
+      "arm64"
+    else
+      throw "unsupported platform";
+in
+
+stdenv.mkDerivation {
+  name = "kryoflux";
+  src = fetchurl {
+    url = "https://www.kryoflux.com/download/kryoflux_3.50_linux_r2.tar.gz";
+    hash = "sha256-qGFXu0FkmCB7cffOqNiOluDUww19MA/UuEVElgmSd3o=";
+  };
+  nativeBuildInputs = [
+    makeWrapper
+    autoPatchelfHook
+  ];
+  buildInputs = [
+    fmt_9
+    libusb1
+  ];
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/java
+    cp -r {docs,testimages,schematics} $out/share
+    cp dtc/kryoflux-ui.jar $out/share/java
+    makeWrapper ${jre}/bin/java $out/bin/kryoflux-ui \
+      --add-flags "-jar $out/share/java/kryoflux-ui.jar" \
+      --set PATH "$out/bin"
+    tar -C $out -xf dtc/${archdir}/kryoflux-dtc*.tar.gz \
```

And then replace this with 
```suggestion
    tar -C $out -xf dtc/${stdenv.hostPlatform.linuxArch}/kryoflux-dtc*.tar.gz \
```